Add FAQ to readme (#34)
ai/check-dts: README.md

$ npx check-dts **/*.tsx?
$ npx check-dts **/*.ts !**/index.ts
```

## FAQ

> ### Does Autoprefixer polyfill Grid Layout for IE?

Reviewer: You forgot to change the question :)

Author: haha! Yes I fixed it

Please make sure that your tsconfig.json doesn't include `skipLibCheck: true`.

> Becuase with this [option](https://www.typescriptlang.org/tsconfig#skipLibCheck) all d.ts-files will be ignored.

Reviewer: Keep 80 columns line. For instance, by moving link’s URL under the text (check Autoprefixer docs).

Author: done! :ok_hand:

$ npx check-dts **/*.tsx?
$ npx check-dts **/*.ts !**/index.ts
```

## FAQ

### It seems that check-dts ignores all d.ts-files. How can I fix this?

> Please make sure that your tsconfig.json doesn't include `skipLibCheck: true`.

Reviewer: ```suggestion
Please make sure that your tsconfig.json doesn’t include `skipLibCheck: true`.
```

Becuase with this [option](https://www.typescriptlang.org/tsconfig#skipLibCheck)

> all d.ts-files will be ignored.

Reviewer: ```suggestion
all `d.ts`-files will be ignored.
```

### Can I use check-dts as a simple validator for d.ts-files?

Yes you can. But note if you have the typescript in your project

> you can validate whether d.ts-files have some issues with the following command:

Reviewer: ```suggestion
you can validate whether `d.ts`-files have some issues with the following command:
```
